pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -0,0 +1,368 @@
+/* hunspell dictionaries */
+
+{ stdenv, fetchurl, unzip }:
+
+let
+  mkDictFromRedIRIS =
+    { shortName, shortDescription, dictFileName, src }:
+    stdenv.mkDerivation rec {
+      inherit src;
+      version = "0.7";
+      name = "hunspell-dict-${shortName}-rediris-${version}";
+      meta = {
```

Comment:
+ You can avoid some typing using, `meta = with stdenv.lib; {`
+ No maintainer field